### ISSUE TITLE
New term: Hyperlordotic gait

### ISSUE THREAD

**Issue by CorneliusDreyling** [obophenotype/human-phenotype-ontology#10802]
**Preferred term label:**
Hyperlordotic gait

**Synonyms**
--

**Definition (free text, please give PubMed ID)**
Distinctive swayback posture and gait pattern with forward tilt of the pelvis and increased lumbar curvature. Typically seen in patients with muscular dystrophy due to axial and core muscle weakness. (DOI: https://doi.org/10.1007/978-3-031-44009-0_4)

**Parent term (use hpo.jax.org/app)**
Gait disturbance HP:0001288

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
FSHD (orphanetcode: 269)

**Your nano-attribution (ORCID)**
0009-0004-9942-3886

**Comment by pnrobinson** [comment]
@CorneliusDreyling this is a difficult subject. Here is a classification of functional gait abnormalities,
https://www.neurology.org/doi/10.1212/WNL.0000000000009649
I am not sure if hyperlordotic gait fits in here? How commonly is this used?

**Comment by pnrobinson** [comment]
@CorneliusDreyling 
Coming back to this I think my initial skepsis was wrong, I have found many usages of the term and am adding it.

Adding new term: 
- ID: HP:6001360
- label: Hyperlordotic gait
- parents: [Gait disturbance (HP:0001288)](https://hpo.jax.org/app/browse/term/HP:0001288)
- pmid: PMID:11166162|PMID:30713909
- def: A gait abnormality characterized by excessive lumbar lordosis (swayback posture, i.e., an inward curvature of the lower spine) and gait pattern with forward tilt of the pelvis and increased lumbar curvature. Hyperlordotic gait is often as a compensatory mechanism for hip extensor weakness, proximal muscle weakness, or hip flexion contracture.
- comment: Typically seen in patients with muscular dystrophy due to axial and core muscle weakness.
- orcid: orcid:0009-0004-9942-3886
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10802
- synonyms: Gait with exaggerated lumbar lordosis(NONE)
